contracts/protocol/facets/DisputeResolverHandlerFacet.sol

```diff
@@ -409,11 +415,13 @@ contract DisputeResolverHandlerFacet is IBosonAccountEvents, ProtocolBase {
      * - Number of DisputeResolverFee structs in array exceeds max
      * - Number of DisputeResolverFee structs in array is zero
      * - DisputeResolverFee array contains duplicates
+     * - Fee amount is a non-zero value. Protocol doesn't yet support fees for dispute resolvers
      *
      * @param _disputeResolverId - id of the dispute resolver
      * @param _disputeResolverFees - list of fees dispute resolver charges per token type. Zero address is native currency. See {BosonTypes.DisputeResolverFee}
+                                     feeAmount will be ignored because protocol doesn't yet support DR fees but DR still needs to provide array of fees to choose supported tokens
      */
-    function addFeesToDisputeResolver(uint256 _disputeResolverId, DisputeResolverFee[] calldata _disputeResolverFees)
+    function addFeesToDisputeResolver(uint256 _disputeResolverId, DisputeResolverFee[] memory _disputeResolverFees)
```

Comment:
I assume the answer is the same as for `createDisputeResolver`, but why does `_disputeResolverFees`  change from `calldata` to `memory`?

Reply:
fixed